### ISSUE TITLE
Why only button tag?

### ISSUE THREAD

**Issue by afc163** [nihey/react-clipboard#10]
Could it render other tag like span or div?

```html
      <Clipboard component="div" data-clipboard-text="I'll be copied">
        copy to clipboard
      </Clipboard>
```

**Comment by nihey** [comment]
Sure, I haven't thought about that when the project was conceived, it'll be added shortly

**Comment by nihey** [comment]
@afc163 Thank you for the input into making this component better :). This should be available on [`0.1.5`](https://github.com/nihey/react-clipboard/releases/tag/0.1.5)

**Comment by afc163** [comment]
:smile:
